conf/messages

```diff
@@ -1065,6 +1065,7 @@ disposalDate.agent.title=When the property was disposed of, what date were contr
 disposalDate.trust.title=When the property was disposed of, what date were contracts exchanged?
 disposalDate.capacitor.title=When the property was disposed of, what date were contracts exchanged?
 disposalDate.personalRep.title=When the property was disposed of, what date were contracts exchanged?
+disposalDate.personalRepInPeriodOfAdmin.title=When the property was disposed of, what date were contracts exchanged?
```

Comment:
it'd be better to update the disposal date and completion date templates to use this user key for period of admin as well - otherwise, if we ever change this message it'll update the cya page only and not the actual page